pkg/selectors/kernel.go

```diff
@@ -363,7 +363,48 @@ func writeMatchValuesInMap(k *KernelSelectorState, values []string, ty uint32) e
 			return fmt.Errorf("Unknown type: %d", ty)
 		}
 		m[val] = struct{}{}
+	}
+	// write the map id into the selector
+	WriteSelectorUint32(k, mid)
+	return nil
+}
 
+func writeMatchRangesInMap(k *KernelSelectorState, values []string, ty uint32) error {
```

Comment:
We seem to, now, have two functions doing very similar things: `writeMatchRangesInMap` and `writeMatchValuesInMap`. Is there a reason we we can't use just one for both cases? I think it would simplify the code a lot.

Reply:
I was leaving the choice, but yep I can remove writeMatchValuesInMap.